superset-frontend/webpack.config.js

```diff
@@ -383,7 +383,9 @@ const config = {
             loader: 'less-loader',
             options: {
               sourceMap: isDevMode,
```

Comment:
Tangential discussion, but it seems like we should just leave sourcemaps on all the time... browsers (at least Chrome) don't download them unless the dev tools are open, so I don't think there' a downside.

Reply:
Just opened up [a PR](https://github.com/apache/superset/pull/19710) to take a quick shot at this.